GW2EIEvtcParser/EncounterLogic/Raids/W7/PeerlessQadim.cs

```diff
@@ -50,26 +51,56 @@ public PeerlessQadim(int triggerID) : base(triggerID)
             EncounterID |= 0x000003;
         }
 
+        protected override List<int> GetTargetsIDs()
+        {
+            return new List<int>()
+            {
+                (int)ArcDPSEnums.TargetID.PeerlessQadim,
+                (int)ArcDPSEnums.TrashID.FriendlyPeerlessQadimPylon,
+                (int)ArcDPSEnums.TrashID.EntropicDistortion,
+            };
+        }
+
         protected override List<ArcDPSEnums.TrashID> GetTrashMobsIDs()
         {
             return new List<ArcDPSEnums.TrashID>()
             {
-                ArcDPSEnums.TrashID.FriendlyPeerlessQadimPylon,
                 ArcDPSEnums.TrashID.HostilePeerlessQadimPylon,
-                ArcDPSEnums.TrashID.EntropicDistortion,
                 ArcDPSEnums.TrashID.BigKillerTornado,
                 ArcDPSEnums.TrashID.EnergyOrb,
+                ArcDPSEnums.TrashID.Brandstorm,
+                ArcDPSEnums.TrashID.GiantQadimThePeerless,
+                ArcDPSEnums.TrashID.DummyPeerlessQadim,
```

Comment:
Who is that? I see it is using the generic red dot as icon.

Reply:
The brandstorm spawns the orange lightning aoes during the entire fight, it's not visible so i gave it a generic red dot, the dummy qadim should be the one spawning the orbs

Reply:
Does it make sense to show them on CR if they are not visible nor interactable? Wouldn't that be confusing?

Reply:
I can comment them out, they should be hidden under this dot (which is qadim's dot i think, it's visible even if i comment brandstorm and dummy)

![6152](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/ae45ecc1-4c3c-4ec1-8405-496d1740022e)

Reply:
I don't see that red dot in current, if you can still see that after commenting brandstorm and dummy, what is it?

Reply:
Ah adding them to GetTrashMobsIDs adds the dot automatically, i was commenting it in ParserIcons. The dot is the brandstorm, I can't see where the dummy Qadim would be at all

Reply:
Technically also the aura pylons only show up last few % of the fight, but they aren't something you can hit or see and they were the only ones to have an icon till now

![6153](https://github.com/baaron4/GW2-Elite-Insights-Parser/assets/23245218/27210839-f000-424a-9e85-1a0eb7332926)

Reply:
Well they were wrongly named, it wouldn't surprise me if the situation around them was simply mis identified. I agree that with the new information in hand we shouldn't show the aura one.

Reply:
Preferences on the icons? Posted them in the other convo

Reply:
The blue neutral state one I'd say.